server/bin/pbench-sync-satellite.sh

```diff
@@ -250,10 +250,30 @@ for host in $hosts ;do
         for x in $state_list; do
             echo "$host/TO-SYNC/$x" >> ${state_change_log}
         done
+        rm $logdir/$host/ok-checks.log
     fi
+
+    if [[ ! -s $logdir/$host/fail-checks.log ]] ; then
+        rm $logdir/$host/fail-checks.log
+    fi
+
+    if [[ -z $(grep -E -v 'OK|FAILED' $logdir/$host/md5-checks.log) ]] ; then
+        rm $logdir/$host/md5-checks.log
+    fi
+
     popd > /dev/null 2>&4
 done
 
+if [[ -s ${logdir}/mv.log ]] ; then
```

Comment:
Can we consider moving this check and removal of `mv.log` to the else block of [line 69](https://github.com/distributed-system-analysis/pbench/pull/1354/files#diff-2b119fd9bfaa54cda0c0fbdad8ae2257R69)?